independent installation of elyra-python-editor-extension is failing with Module not found

**Describe the issue**
While trying to install **elyra-python-editor-extension** independently, by following the documentation for installing the extension independently of elyra https://elyra.readthedocs.io/en/latest/getting_started/installation.html#pip, the process errored out with Module not found error during the `jupyter lab build` step. 

` ModuleNotFoundError: Module not found: Error: Can't resolve '@material-ui/lab' in '/opt/app-root/share/jupyter/lab/staging/node_modules/@elyra/ui-components/lib'`
is the @material-ui/lab package missing ?
what should i do to resolve this issue? i couldn't find any similar issue.

**To Reproduce**
Steps to reproduce the behavior:
1. Open the container: `podman run -it quay.io/thoth-station/s2i-minimal-py38-notebook:latest bash`
2. Install: `pip install elyra-python-editor-extension`
3. Execute: `jupyter lab build`
4. See error log present in /tmp dir

**Screenshots or log output**
Small snippet of the lab build log:
```
[LabBuildApp] yarn run v1.21.1
$ webpack --config webpack.prod.minimize.config.js
[webpack-cli] ModuleNotFoundError: Module not found: Error: Can't resolve '@material-ui/lab' in '/opt/app-root/share/jupyter/lab/staging/node_modules/@elyra/ui-components/lib'
    at /opt/app-root/share/jupyter/lab/staging/node_modules/webpack/lib/Compilation.js:1672:28
    at /opt/app-root/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:712:13
    at eval (eval at create (/opt/app-root/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /opt/app-root/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:273:22
    at eval (eval at create (/opt/app-root/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /opt/app-root/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:402:22
    at /opt/app-root/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:117:11
    at /opt/app-root/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:628:24
    at /opt/app-root/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:782:8
    at /opt/app-root/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:902:5
resolve '@material-ui/lab' in '/opt/app-root/share/jupyter/lab/staging/node_modules/@elyra/ui-components/lib'
  Parsed request is a module
```
full error: 
[lab.log](https://github.com/elyra-ai/elyra/files/6308310/lab.log)

**Deployment information**
Describe what you've deployed and how:
 - **elyra-python-editor-extension** version: 2.2.1
 - Operating system: linux , ubi8 image
 - Installation source: PyPI
 - Deployment type: Docker image
 
```
(app-root) python --version
Python 3.8.3
(app-root) node --version
v14.16.1
(app-root) npm --version
6.14.12
 (app-root) jupyter --version
jupyter core     : 4.7.1
jupyter-notebook : 6.3.0
qtconsole        : not installed
ipython          : 7.22.0
ipykernel        : 5.5.3
jupyter client   : 6.1.12
jupyter lab      : 3.0.14
nbconvert        : 5.6.1
ipywidgets       : 7.6.3
nbformat         : 5.1.3
traitlets        : 5.0.5
```

Additional Context:
Related-to: https://github.com/thoth-station/s2i-generic-data-science-notebook/pull/20

--- a/packages/ui-components/package.json
+++ b/packages/ui-components/package.json
@@ -33,6 +33,7 @@
   "dependencies": {
     "@jupyterlab/apputils": "^3.0.0",
     "@jupyterlab/ui-components": "^3.0.0",
+    "@material-ui/lab": "^4.0.0-alpha.18",
     "@material-ui/core": "^4.1.2",
     "@material-ui/icons": "^4.2.1",
     "react": "^17.0.1",

